submissions/7SergSmith7/HTML-CSS-Popup/styles.css

```diff
@@ -0,0 +1,144 @@
+a {
+  text-decoration: none;
+  color: black;
+}
+a:focus {
+  color: gray;
+  border: 1px solid gray;
+}
+a:hover {
```

Comment:
Avoid content shifting on hover, except it necessary.
In your case content shifting must be removed, because it looks no good.